Update Access Control based on review comments (#1794)
emqtt/emq-docs-cn: en_US/access-control/authn/authn.md

> 1. If the username is the same, and:

Reviewer: ```suggestion
1. If authentication credentials exist, and:
```

>    - the authentication information matches, the client will be allowed to connect.

Reviewer: ```suggestion
   - the authentication information matches(e.g. password is correct, JWT is valid), the client will be allowed to connect.
```

>    - the authentication information does not match, the client will not be allowed to connect.

Reviewer: ```suggestion
   - the authentication information does not match, and the client will be denied to connect.
```

> 1. If multiple authenticators are configured, and EMQX fails to retrieve the information from the current authenticator, it will:

Reviewer: Sequential lookup is a key information

```suggestion
2. When multiple authenticators are configured, EMQX will look for credentials in order. Once the match is successful it will allow the client to connect.
If no credentials are found in the current authenticator, it will:
```

   - continue to retrieve the information from other authenticators.
   - refuse the connection if this is already the last authenticator.

::: tip 

> Please note that malfunctioning authentication is also treated as "Not Found".

Reviewer: ```suggestion
The current authenticator will also be skipped when the authenticator is in a disabled state or there are errors in the process of authentication, for example, the database is not available.
```
